clang/lib/CIR/CodeGen/CIRGenTypes.cpp

```diff
@@ -743,6 +743,10 @@ mlir::Type CIRGenTypes::ConvertType(QualType T) {
   return ResultType;
 }
 
+namespace cir {
+void computeSPIRKernelABIInfo(CIRGenModule &CGM, CIRGenFunctionInfo &FI);
```

Comment:
This should be in the appropriated header, not here.

Reply:
Sure! Referencing it from `TargetInfo.h` now.